Description: adjust ejabberdctl
 The default ejabberdctl needs some changes to work correctly and to
 avoid unnecessary problematic changes compared to older versions.
Author: Philipp Huebner <debalance@debian.org>

--- a/ejabberdctl.template
+++ b/ejabberdctl.template
@@ -13,7 +13,7 @@ ERLANG_NODE=ejabberd@localhost
 ERL="{{erl}}"
 IEX="{{bindir}}/iex"
 EPMD="{{epmd}}"
-INSTALLUSER={{installuser}}
+INSTALLUSER=ejabberd

 # check the proper system user is used if defined
 EXEC_CMD="false"
